app/controllers/invoices_controller.rb

```diff
@@ -7,6 +7,8 @@ def index
 
   def show
     @invoice = Invoice.find(params[:invoice_id])
+    @items = @invoice.items
```

Comment:
great for now! Maybe in the future, we work this down to just one instance variable